alpha/lib/model/categoryEntryPeer.php

```diff
@@ -195,6 +195,15 @@ public static function selectByEntryId($entryId)
 		
 		return self::doSelect($c);
 	}
+
+	public static function selectByEntryIds(array $entryIds, $privacyContext)
+	{
+		$c = new Criteria();
+		$c->add(self::ENTRY_ID, $entryIds, Criteria::IN);
+		$c->add(self::PRIVACY_CONTEXT, $privacyContext, Criteria::EQUAL);
```

Comment:
If we don't have privacy_context, the query searches only for entries without.

Reply:
yes, but nothing to do about it. this is how it worked before with on entryId. KMS will always use privacy context in any case